.gitignore

```diff
@@ -51,6 +51,7 @@ config/database.yml
 
 vcr.log
 .byebug_history
+config/app_environment_variables.rb
```

Comment:
I checked it in with the keys for the accounts I created to test the code during development. I wanted to leave these in source control since the tests depend on them, and to ensure the calls to the API won't break during future development.

I then added this file to .gitignore because we never want to accidentally check in the real API keys to production.

Is there a better way to handle this situation?